rescript

```diff
@@ -222,9 +222,12 @@ if (
   }
 
   verbose = delegate_args.includes("-verbose");
-
+  /**
+   * @type {child_process.ChildProcess}
+   */
+  var p;
   try {
-    child_process.execFileSync(bsb_exe, delegate_args, { stdio: "inherit" });
+    p = child_process.spawn(bsb_exe, delegate_args, { stdio: "inherit" });
```

Comment:
this is made async so that we can do some clean up(releaseLock) after it is done later